Backend/ChatAPI.Data/ChatAppDbContext.cs

```diff
@@ -0,0 +1,17 @@
+namespace ChatAPI.Data;
+using Microsoft.EntityFrameworkCore;
+
+public class ChatAppDbContext : DbContext
+{
+
```

Comment:
Add a constructor, it should be accepting DbContextOptions parameter. Pass that param to base class.